java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -4005,7 +4005,7 @@ void testReplaceLeafNodeInList() {
 
   @Test
   void testReplaceLeafNodeInListWithIllegal() {
-    assertThrows(IllegalArgumentException.class, () -> {
+    assertThrows(AssertionError.class, () -> {
```

Comment:
can we also assert on the content of the expected exception message similar to https://stackoverflow.com/questions/40268446/junit-5-how-to-assert-an-exception-is-thrown.

Reply:
I have updated. Can you see the change?

Reply:
you may not have pushed yet @razajafri